src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -408,28 +378,22 @@ void medRegistrationSelectorToolBox::enableSelectorToolBox(bool enable){
     this->setEnabled(enable);
 }
 
-void medRegistrationSelectorToolBox::onJobAdded(medJobItem* item, QString jobName){
-    if (d->process)
-        if (jobName == d->process->identifier()){
-            dtkAbstractProcess * proc = static_cast<medRunnableProcess*>(item)->getProcess();
-            if (proc==d->process)
-                enableSelectorToolBox(false);
-        }
-}
-bool medRegistrationSelectorToolBox::setFixedData(medAbstractData* data)
+void medRegistrationSelectorToolBox::setFixedData(medAbstractData* data)
 {
     d->fixedData = data;
-
-    return d->undoRedoProcess
-            && d->undoRedoProcess->setFixedInput(d->fixedData)
-            && d->undoRedoProcess->setMovingInput(d->movingData);
+    if(d->undoRedoProcess)
+    {
+        d->undoRedoProcess->setFixedInput(d->fixedData);
+        d->undoRedoProcess->setMovingInput(d->movingData);
+    }
```

Comment:
The refactoring was unnecessary though, since there was useless code in the first place which, if removed, resulted in no duplication:
```d->undoRedoProcess->setFixedInput(d->fixedData)``` only needs to be done in ```setFixed```
```d->undoRedoProcess->setMovingInput(d->movingData)``` only needs to be done in ```setMoving```